### DISCUSSION TITLE
Can I use evolve.py to finetune on own dataset (i.e. not only pedestrian?)

### DISCUSSION THREAD

**Discussion by ulrikabremberg** [mikel-brostrom/yolo_tracking#845] [General]
Hi! I've been trying to use the evolve.py script to finetune the tracker configs on my own annotated dataset, but I can't get it to work. I have created the MOT like structure like described, with image folder and gt.txt annotations etc. In the val_utils folder I have adapted the classes in mot_challenge_2d_box.py to match my classes. The script generates accurate predictions in the "exp" output folder, but upon running the run_mot_challenge it seems to crash. Is it at all possible to use this script if not interested in pedestrians, but rather the coco classes?

I've been looking at these issues before: #590,  and also this one: #669 that seem to have a similar problem but not any direct solutions to my problem.

This is the output:
![image](https://user-images.githubusercontent.com/47598668/236858648-2b33c872-13e8-4edf-8df3-e6611925d263.png)

Hope it makes sense :)

**Comment by ulrikabremberg** [comment]
Never mind, forgot to remove the class labels and thats what messed it up I think :)

**Reply by mikel-brostrom** [reply]
Hej! Did you solve it? Looks like you are missing the `tabulate` package as well

**Reply by ulrikabremberg** [reply]
Yes, all solved now after setting all gt’s and preds to pedestrian class ☺️ Thanks for a great repo!
